raiden-ts/src/raiden.ts

```diff
@@ -41,18 +42,19 @@ import {
 } from 'rxjs/operators';
 
 import './polyfills';
-import { TokenNetworkRegistry } from './contracts/TokenNetworkRegistry';
-import { TokenNetwork } from './contracts/TokenNetwork';
-import { HumanStandardToken } from './contracts/HumanStandardToken';
-
-import TokenNetworkRegistryAbi from './abi/TokenNetworkRegistry.json';
-import TokenNetworkAbi from './abi/TokenNetwork.json';
-import HumanStandardTokenAbi from './abi/HumanStandardToken.json';
```

Comment:
Looks like we don't need `src/abi/` anymore? If yes, might make sense to remove the `abi` related code from `scripts/copyContracts.js:preBuild()` as well unless we need it in the future

Reply:
We still need the abis to build the contracts with typechain. We just don't need it at runtime. Unless typechain gained the ability to parse the bundled contracts.json, we either keep it, or remove it after running typechain, but it still needed.